.github/workflows/cd.yml

```diff
@@ -2,8 +2,8 @@ name: Continuous Delivery
 on:
   push:
     branches:
-      - master
-      - cloud
+      - main
+      - legacy
```

Comment:
This should be removed, we should not run CD pipeline on legacy app (it will fail).

Reply:
👍